### ISSUE TITLE
EKS: multus unable to start

### ISSUE THREAD

**Issue by brdji** [testground/testground#1457]
Issue - multus suddenly unable to start, with the following error message:
```
"Error: failed to start container "generate-kubeconfig": Error response from daemon: OCI runtime create failed: container_linux.go:380: starting container process caused: exec: "/usr/src/multus-cni/bin/generate-kubeconfig": stat /usr/src/multus-cni/bin/generate-kubeconfig: no such file or directory: unknown"
```
Solution - multus code updated upstream, so we had to change the image our deployments are pulling

_Originally posted by @brdji in https://github.com/testground/testground/issues/1435#issuecomment-1240890745_

**Comment by dektech** [comment]
Commenting per discussion in the infra PR:

This was caused by a complete overhaul in multus logic. In v3.9.1 they started migrating to the client/server model.
This new deployment is called 'thick plugin', in contrast to the previous deployment, which is now called 'thin plugin'.

So once we saw the changes in the documentation, we changed the names of the images our version of multus was pulling and it deployed correctly again.
